client/transport/tlsconfig_clone_go16.go

```diff
@@ -0,0 +1,13 @@
+// +build go1.6
+// +build !go1.7
```

Comment:
can you move these build tags to the same line?  or why is this needed at all?  isn't it enough to say only build for `go1.6` ?